doc/example/exampleBatchFile.slurm

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
This file is not referenced anywhere, so chances are low that it will be helpful to anybody. I'd rather include it verbatim in the respective module docstring.

Reply:
you mean in MPIPoolEngine() docstring?

Reply:
I meant a file-level docstring, but the class documentation is good as well.

Reply:
moved it to MPIPoolEngine